spiceaidocs/docs/components/data-accelerators/arrow.md

```diff
@@ -0,0 +1,34 @@
+---
+title: 'In-Memory Arrow Data Accelerator'
+sidebar_label: 'In-Memory Arrow Data Accelerator'
+description: 'In-Memory Arrow Data Accelerator Documentation'
+sidebar_position: 1
+---
+
+The In-Memory Arrow Data Accelerator is the default data accelerator in Spice. It uses Apache Arrow to store data in-memory for fast access and query performance.
+
+## Configuration
+
+To use the In-Memory Arrow Data Accelerator, specify `arrow` as the `engine` for acceleration.
+
+```yaml
+datasets:
+  - from: spice.ai:path.to.my_dataset
+    name: my_dataset
+    acceleration:
+      engine: arrow
+```
+
+## Limitations
+
+- The In-Memory Arrow Data Accelerator does not support persistent storage. Data is stored in-memory and will be lost when the Spice runtime is stopped.
+- The In-Memory Arrow Data Accelerator does not support schemas with nested arrays/lists, UTF8/string arrays/lists, structs, nested structs, or map fields.
```

Comment:
Is this true? I thought the Arrow accelerator supported all possible Arrow types

Reply:
Removed the following limitation and will merhe this PR. We will be able to add this back if needed.

```
- The In-Memory Arrow Data Accelerator does not support schemas with nested arrays/lists, UTF8/string arrays/lists, structs, nested structs, or map fields.
```